openoa/analysis/wake_losses.py

```diff
@@ -717,6 +927,9 @@ def run(
 
             df_wd_bin = self.aggregate_df_sample.groupby("wind_direction_bin").sum()
 
+            index = np.arange(0.0, 360.0, self.wd_bin_width_LT_corr).tolist()
```

Comment:
I don't think `index` needs to be converted to a list since Pandas can handle arrays fairly easily.